### PR TITLE
Mention pytorchbot command on label error

### PR DESCRIPTION
**Pull request by kit1980**
It's not clear how to add a label, especially for contributors without write permissions - they don't have a UI for that.

One recent struggle example - https://github.com/pytorch/pytorch/pull/94671

### CODE REVIEW

**Review by PaliC — APPROVED** [review]
lgtm, given its just changing a string, I think this change should be fine.

File: .github/scripts/label_utils.py [hunk 1]
```diff
@@ -24,6 +24,9 @@
 
 If not, please add the `topic: not user facing` label.
```
Comment (janeyx99) [inline .github/scripts/label_utils.py:25]:
```suggestion
If not, please add the `topic: not user facing` label, either through the GitHub UI or by commenting to pytorchbot, for example
`@pytorchbot label "topic: not user facing"`
```
is more concise/less repetitive

Comment (kit1980) [reply .github/scripts/label_utils.py:25]: I don't want to mention UI, because contributors without write access don't have it and get confused.
Agree on more concise wording, I'll rephrase a bit.

**Comment by kit1980** [comment]
> lgtm, given its just changing a string, I think this change should be fine.

There is potential issue that pytorchbot mention will trigger pytorchbot.

**Comment by malfet** [comment]
If your changes are user facing and intended to be a part of release notes, please use a label starting with `release notes:`.

If not, please add the `topic: not user facing` label.

To add a label, you can comment to pytorchbot, for example
`@pytorchbot label "topic: not user facing"`

For more information, see
https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work.

**Comment by malfet** [comment]
> @pytorchbot help

**Review by malfet — APPROVED** [review]

**Review by seemethere — APPROVED** [review]
